nmostesting/mocks/Node.py

```diff
@@ -182,23 +231,36 @@ def patch_staged(self, resource, resource_id, request_json):
         response_data = deepcopy(activations['staged'])
         response_code = 200
 
-        # Check transport params against constraints
-        if request_json.get('transport_params'):
-            transport_params = request_json['transport_params']
-            constraints = _get_constraints(resource)
-
-            for key, value in constraints.items():
-                if key in transport_params[0]:
-                    if value:
-                        # There is a constraint for this param and a value in the request to check
-                        check = _check_constraint(value, transport_params[0][key])
-                        if not check:
-                            response_data = {'code': 400, 'debug': None,
-                                             'error': 'Transport param {} does not satisfy constraints.'.format(key)}
-                            response_code = 400
-                            return response_data, response_code
-                    # No constraint to check or passed check so save param from request
-                    response_data['transport_params'][0][key] = transport_params[0][key]
+        # NOTE that this Receiver only has a single leg (no ST 2022-7 redundancy)
+        # Copy SDP parameters into transport_params in response
+        if 'transport_file' in request_json:
+            sdp_params = self.parse_sdp(request_json['transport_file'])
```

Comment:
As @KonradDuda identified, this needs to handle `transport_file` being present, but setting its `data` to `null`.

Reply:
Thanks for testing this @KonradDuda - yes agreed @garethsb let's guard for `null`